src/assets/stylesheets/shared/_layout.scss

```diff
@@ -237,8 +237,7 @@ footer
 }
 
 .btn{
-  min-width: 90px;
-  font-size: $font-size-base;
```

Comment:
font size base should be in global styles if we want it but min-width:90px is breaking the UI